### DISCUSSION TITLE
basic-extra-meal/deep-copy

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#208] [Announcements]
# basic-extra-meal/deep-copy

本文详细讨论了Java中的浅拷贝和深拷贝概念，解析了它们如何在实际编程中应用。文章通过实例演示了如何实现浅拷贝与深拷贝，以帮助读者更好地理解这两种拷贝方式在Java编程中的作用与应用场景。

https://tobebetterjavaer.com/basic-extra-meal/deep-copy.html

**Comment by SlimYXin** [comment]
照着敲的输出结果不一样啊，浅拷贝后writer1和writer2是一个地址，也没有变更
